### ISSUE TITLE
Visual studio 2019 (16.3.1) Compilation issue

### ISSUE THREAD

**Issue by Gazolik** [steinbergmedia/vst3sdk#39]
Visual studio professional 2019 16.3.1
MSVC 14.23.28105

Works with MSVC 14.22.27905.
A new deprecation warning has been added to MSVC, defined in <experimental/filesystem>:

```
#ifndef _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING
#error The <experimental/filesystem> header providing std::experimental::filesystem is deprecated by Microsoft \
and will be REMOVED. It is superseded by the C++17 <filesystem> header providing std::filesystem. \
You can define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING to acknowledge that you have received this warning.
#endif // _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING
```

So the `public.sdk/source/vst/hosting/module_win32.cpp` can't compile.

**Comment by Gazolik** [comment]
My bad, no error on the last sdk.

**Comment by olilarkin** [comment]
same error here

**Comment by MirkoCovizzi** [comment]
I solved by adding the preprocessor definition "_SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING" to the validator project.

**Comment by scheffle** [comment]
Should be solved now with the 3.6.14 release.
